Jenkinsfile

```diff
@@ -1,29 +1,29 @@
 pipeline {
```

Comment:
The modification in this file are related to the same issue mentioned in the description of this PR? If not, please explain this specific changes in the Jenkinsfile.

Reply:
The modification in the Jenkinsfile is just in the indentation, because he use a curly braces to define scope, but the PEP8 pattern require 4 multiples in the indentation.